src/Node/Render.php

```diff
@@ -44,9 +68,12 @@ protected function addTemplateArguments(Twig_Compiler $compiler) {
   }
 
   /**
-   * Returns the component context from the component config file.
+   * Returns the component context variables from its config file.
    *
    * @todo Merge specific keys to prevent config duplication.
+   *
```

Comment:
why we are not achieving this `@todo` right now?

Reply:
Created an issue https://github.com/netzstrategen/drupal-twig_fractal/issues/3.